### PR TITLE
SLM-81 Add integration tests

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: assets/js/page-enhancements.js [hunk 1]
```diff
@@ -18,9 +18,19 @@
...
+})($, document)
+;(autoFocusContactPrisonNameField = ($, document) => {
+  $(document).ready(() => {
+    $('#create-new-contact-by-prison-number-form #prisonerName').focus()
+  })
+})($, document)
+;(autoFocusPrisonerDobField = ($, document) => {
+  $(document).ready(() => {
+    $('#create-new-contact-by-prisoner-name-form #prisonerDob-day').focus()
   })
```
Comment (timharrison-moj) [inline assets/js/page-enhancements.js:34]: Out of interest, does this autofocus approach work with IE? If so, I might steal it.

File: server/filters/recipientTableRowsFilter.ts [hunk 2]
```diff
@@ -9,7 +10,9 @@ export default function recipientTableRowsFilter(recipients: Array<Recipient>):
   return recipients.map((recipient, idx) => {
     return Array.of(
       { text: recipient.prisonerName }, // Column 1 - Prisoner Name
-      { text: recipient.prisonNumber }, // Column 2 - Prison number or DOB - TODO SLM-81
+      {
+        text: recipient.prisonNumber ? recipient.prisonNumber : moment(recipient.prisonerDob).format('DD-MM-YYYY'), // Column 2 - Prison number or DOB
+      },
```
Comment (timharrison-moj) [inline server/filters/recipientTableRowsFilter.ts:15]: Can prisoner DOB ever be absent / null?

**Review by timharrison-moj — APPROVED** [review]
LGTM. Minor questions only.

File: server/filters/recipientTableRowsFilter.ts (hunk 2 above)
Comment (mikehalmamoj) [reply server/filters/recipientTableRowsFilter.ts:15]: Apparently it does: https://caniuse.com/autofocus

File: assets/js/page-enhancements.js (hunk 1 above)
Comment (mikehalmamoj) [reply assets/js/page-enhancements.js:34]: Apparently it does: https://caniuse.com/autofocus

File: server/filters/recipientTableRowsFilter.ts (hunk 2 above)
Comment (mikehalmamoj) [reply server/filters/recipientTableRowsFilter.ts:15]: Yes we capture prisoner name and either prison ID or prisoner DOB. With a separate form for each method bizarrely :shrug:

File: assets/js/page-enhancements.js (hunk 1 above)
Comment (mikehalmamoj) [reply assets/js/page-enhancements.js:34]: Ooh sorry that was the HTML focus attribute - which we had mixed results with. This gives a good breakdown of the various ways of focusing and compatibility: https://caniuse.com/?search=focus

**Pull request merged**